### DISCUSSION TITLE
Preexisting Conditions on ICU Admission

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1449] [MIMIC-IV]
Hello Mimic-Community, me and my team are working on some questions regarding organ dysfunction on the ICU. One parameter of high interest for us are preexisting conditions that are already manifest in a patient on ICU admission. I was thinking about using coded diagnoses in Mimic-ED for that purpose or checking for medications on admission. I was wondering if there are any different approaches that I did not yet think about? Are there any best practices in place?

Kind regards

**Comment by marymlucas** [comment]
Hi @aegis301, this is not an official response, just a regular user perspective. Since you mentioned MIMIC-ED I assume you're using MIMIC-IV.  Your approach sounds solid since it's fair to assume most patients will first come through the ED, but there may be some direct admits or transfers from outside hospitals bypassing the ED, so you may miss some data.  

An additional step I would try is if you're interested in a particular ICU admission for a patient is to check for the diagnoses for their previous admissions. The  mimiciv_hosp module has a list of patient icd diagnosis codes with their respective hadm_id so you can match diagnoses to admissions. 

Also, a bit off topic I believe this kind of question best belongs in Discussions, not Issues.  See [https://github.com/MIT-LCP/mimic-code/discussions/1087](https://github.com/MIT-LCP/mimic-code/discussions/1087)

Mary

**Comment by aegis301** [comment]
Thank you so much for your kind reply @marymlucas , I will immediately move this topic to the discussion board, sorry for the inconvenience. 

I think you're right we should take into account that many patients on the ICU did not go through the ED, especially patients that are there for postoperative observation after elective procedures. Great advice with the previous admissions we will definately try that!
